lib/data/monorepo.json

```diff
@@ -571,6 +571,7 @@
     "apache-camel": "/^org.apache.camel:/",
     "aws-java-sdk": "/^com.amazonaws:aws-java-sdk-/",
     "aws-java-sdk-v2": "/^software.amazon.awssdk:/",
+    "axis2": "/^org.apache.axis2:/",
```

Comment:
do they publish the source url? so they can be matched via source url instead of names?

Reply:
yes, they do

https://github.com/apache/axis-axis2-java-core/blob/master/modules%2Fadb-codegen%2Fpom.xml#L41

Reply:
@viceice I initially thought about it, but also thought this would be worse:

1. They could change the source URL in each individual pom.xml to point to the subdirectory of the module, this would break the preset
2. They could change it to GitHub.

I believe the preset is less likely to break if we rely on the group id.

But I can make the changes, you decide. Please let me know.

Reply:
It looks like a source URL prefix of `https://gitbox.apache.org/repos/asf` would be fairly safe, but I think you're right about them possibly changing to github in future. In that case I'm ok with it being based on the prefix

Reply:
we're doing it for almost all others when source url is available

Reply:
Good point

Reply:
Please check again.